imagegw/shifter_imagegw/api.py

```diff
@@ -211,6 +216,79 @@ def pull(system, imgtype, tag):
     return jsonify(create_response(rec))
 
 
+# Import image
+# This will import the requested image from a file path on the system.
+@app.route('/api/doimport/<system>/<imgtype>/<path:tag>/', methods=["POST"])
+def doimport(system, imgtype, tag):
+    """ 
+    Pull a specific image and tag for a systems. 
+    """
+    if imgtype == "docker" and tag.find(':') == -1:
```

Comment:
I think this would be imported with something like custom right?  Maybe we leave out the imgtype all together.

Reply:
I changed this to a check for either docker or custom. We could take out the imgtype check entirely, but it might be better to leave it in in case someone makes some weird new image type in the future.